### DISCUSSION TITLE
StatTypes do not work on non-human character that has characterinfo

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15260] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

.

### Reproduction steps

1. create a humanoid character that has characterinfo
2. equip a slipsuit
3. walking speed is not affected

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by A2ur3** [comment]
and what's the difference between HasCharacterInfo="true" and HasInfo="True"?
I tried both but none of them make StatTypes work

**Comment by Regalis11** [comment]
This has been addressed in our private repo now. Fix coming up in the next unstable build and the upcoming release.
